intf_.mod

```diff
@@ -2275,7 +2277,7 @@ PROCEDURE initwrec () {
     if (num>NSW) { printf("INTF initwrec() WARN: can only store %d ww vecs\n",NSW); hxe();}
     nsw=(double)num;
     for (k=0;k<num;k++) {
-      cap = list_vector_px2(ob, k, &wwo[k], &ww[k]);
+      cap = list_vector_px2(ob, k, &wwo[k], (IvocVect**)ww[k]);
```

Comment:
`ww` should be declared as well as `IvocVect*`?

Reply:
you are right. Done.

Reply:
Wondering whether this is a mistake and we should keep the `&`?